templates/values.go

```diff
@@ -22,12 +22,19 @@ type StepSSH struct {
 var DefaultSSHTemplates = SSHTemplates{
 	User: []Template{
 		{
-			Name:         "include.tpl",
+			Name:         "base_config.tpl",
 			Type:         Snippet,
-			TemplatePath: "templates/ssh/include.tpl",
+			TemplatePath: "templates/ssh/base_config.tpl",
```

Comment:
Can we leave it as `include.tpl` or perhaps `ssh_config.tpl` would be better. Base doesn't tell you anything.